playground/tools/scrap.py

```
@@ -0,0 +1,357 @@
+            "success": True,
+            "goods_no": goods_no,
+            "review_list": improve_review_list
+        }
+        
+    except Exception as e:
+        return {
+            "success": False,
+            "error": str(e),
+        }
+
+def get_promotion_fields(promotion):
```

Ruff (0.12.2)

Missing return type annotation for public function `get_promotion_fields`

(ANN201)